### PR TITLE
 Changes to be committed:

### PR DESCRIPTION
**Pull request by xiaowenjia**
new file:   Autorun_multiqc_stats_collect/README.md
	new file:   Autorun_multiqc_stats_collect/collect_results.sh

### CODE REVIEW

**Review by stschiff — APPROVED** [review]
Wow. This is wonderful, thanks for contributing this! @TCLamnidis what do you say?

File: Autorun_multiqc_stats_collect/collect_results.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
...mageprofiler-std	mtnucratio_mqc-generalstats-mtnucratio-nuc_cov_avg	mtnucratio_mqc-generalstats-mtnucratio-nucreads	QualiMap_mqc-generalstats-qualimap-percentage_aligned	QualiMap_mqc-generalstats-qualimap-total_reads	QualiMap_mqc-generalstats-qualimap-general_error_rate	SexDetErrmine_mqc-generalstats-sexdeterrmine-RateErrX	SexDetErrmine_mqc-generalstats-sexdeterrmine-RateErrY	SexDetErrmine_mqc-generalstats-sexdeterrmine-RateX	SexDetErrmine_mqc-generalstats-sexdeterrmine-RateY	nuclear_contamination_mqc-generalstats-nuclear_contamination-Num_SNPs	nuclear_contamination_mqc-generalstats-nuclear_contamination-Method1_MOM_estimate	nuclear_contamination_mqc-generalstats-nuclear_contamination-Method1_MOM_SE	nuclear_contamination_mqc-generalstats-nuclear_contamination-Method1_ML_estimate	nuclear_contamination_mqc-generalstats-nuclear_contamination-Method1_ML_SE	nuclear_contamination_mqc-generalstats-nuclear_contamination-Method2_MOM_estimate	nuclear_contamination_mqc-generalstats-nuclear_contamination-Method2_MOM_SE	nuclear_contamination_mqc-generalstats-nuclear_contamination-Method2_ML_estimate	nuclear_contamination_mqc-generalstats-nuclear_contamination-Method2_ML_SE"; echo ${hv} > ${of1}
```
Comment (TCLamnidis) [inline Autorun_multiqc_stats_collect/collect_results.sh:9]:
```suggestion
hv="Sample	snp_coverage_mqc-generalstats-snp_coverage-Covered_Snps	snp_coverage_mqc-generalstats-snp_coverage-Total_Snps	Samtools_Flagstat_(pre-samtools_filter)_mqc-generalstats-samtools_flagstat_pre_samtools_filter-flagstat_total	Samtools_Flagstat_(pre-samtools_filter)_mqc-generalstats-samtools_flagstat_pre_samtools_filter-mapped_passed	Samtools_Flagstat_(post-samtools_filter)_mqc-generalstats-samtools_flagstat_post_samtools_filter-flagstat_total	endorSpy_mqc-generalstats-endorspy-endogenous_dna	Samtools_Flagstat_(post-samtools_filter)_mqc-generalstats-samtools_flagstat_post_samtools_filter-mapped_passed	endorSpy_mqc-generalstats-endorspy-endogenous_dna_post	Picard_mqc-generalstats-picard-PERCENT_DUPLICATION	DamageProfiler_mqc-generalstats-damageprofiler-5_Prime1	DamageProfiler_mqc-generalstats-damageprofiler-5_Prime2	DamageProfiler_mqc-generalstats-damageprofiler-3_Prime1	DamageProfiler_mqc-generalstats-damageprofiler-3_Prime2	DamageProfiler_mqc-generalstats-damageprofiler-mean_readlength	DamageProfiler_mqc-generalstats-damageprofiler-median	mtnucratio_mqc-generalstats-mtnucratio-mtreads	mtnucratio_mqc-generalstats-mtnucratio-mt_cov_avg	mtnucratio_mqc-generalstats-mtnucratio-mt_nuc_ratio	QualiMap_mqc-generalstats-qualimap-mapped_reads	QualiMap_mqc-generalstats-qualimap-mean_coverage	QualiMap_mqc-generalstats-qualimap-median_coverage	QualiMap_mqc-generalstats-qualimap-1_x_pc	QualiMap_mqc-generalstats-qualimap-2_x_pc	QualiMap_mqc-generalstats-qualimap-3_x_pc	QualiMap_mqc-generalstats-qualimap-4_x_pc	QualiMap_mqc-generalstats-qualimap-5_x_pc	QualiMap_mqc-generalstats-qualimap-avg_gc	DamageProfiler_mqc-generalstats-damageprofiler-std	mtnucratio_mqc-generalstats-mtnucratio-nuc_cov_avg	mtnucratio_mqc-generalstats-mtnucratio-nucreads	QualiMap_mqc-generalstats-qualimap-percentage_aligned	QualiMap_mqc-generalstats-qualimap-total_reads	QualiMap_mqc-generalstats-qualimap-general_error_rate	SexDetErrmine_mqc-generalstats-sexdeterrmine-RateErrX	SexDetErrmine_mqc-generalstats-sexdeterrmine-RateErrY	SexDetErrmine_mqc-generalstats-sexdeterrmine-RateX	SexDetErrmine_mqc-generalstats-sexdeterrmine-RateY	nuclear_contamination_mqc-generalstats-nuclear_contamination-Num_SNPs	nuclear_contamination_mqc-generalstats-nuclear_contamination-Method1_MOM_estimate	nuclear_contamination_mqc-generalstats-nuclear_contamination-Method1_MOM_SE	nuclear_contamination_mqc-generalstats-nuclear_contamination-Method1_ML_estimate	nuclear_contamination_mqc-generalstats-nuclear_contamination-Method1_ML_SE	nuclear_contamination_mqc-generalstats-nuclear_contamination-Method2_MOM_estimate	nuclear_contamination_mqc-generalstats-nuclear_contamination-Method2_MOM_SE	nuclear_contamination_mqc-generalstats-nuclear_contamination-Method2_ML_estimate	nuclear_contamination_mqc-generalstats-nuclear_contamination-Method2_ML_SE"
echo ${hv} > ${of1}
```

File: Autorun_multiqc_stats_collect/collect_results.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+for P in $(cat $inf); do 
+    tfn1="/mnt/archgen/Autorun_eager/eager_outputs(_old)/TF/*/"${P}"/multiqc/multiqc_data/multiqc_general_stats.txt"
```
Comment (TCLamnidis) [inline Autorun_multiqc_stats_collect/collect_results.sh:11]:
```suggestion
    tfn1="/mnt/archgen/Autorun_eager/eager_outputs/TF/*/"${P}"/multiqc/multiqc_data/multiqc_general_stats.txt"
```
The `eager_outputs_old` directory is not really meant as "live" data, but is just a temporary storage for old results until a new run has finished.

**Review by TCLamnidis — CHANGES_REQUESTED** [review]
script file: 

- [ ] Could you please make the arguments readable from CLI? Each user copying and editing the script will be a hassle. Happy to help with/do that if you want.
- [ ] Include a version number printed out when a user specifies the `-v/--version` option.
- [ ] Perhaps allow users to specify if they want TF or SG results for each individual?

The README should ideally include: 
- [ ] an example of the input sample list file (something simple like below will do: 
```
ABC001
ABC002
ABC003
```
- [ ] An example command that can be used to run the script, like 
 ```
./collect_results.sh target_individuals.txt output_table.txt
 ```

**Review by TCLamnidis — APPROVED** [review]

**Pull request merged**